generative-ai-solutions/bedrock-lambda-layer/lambda_base.py

```diff
@@ -0,0 +1,138 @@
+import boto3
+import base64
+from botocore.exceptions import ClientError
+from pathlib import Path
+import logging
+import json
+import zipfile
+import io
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+layer_arn = 'arn:aws:lambda:us-east-1:141051459377:layer:bedrock-1-28-57:4' 
```

Comment:
lets get rid of these ARNs before merge

Reply:
Pushed a new commit that removed layer and role arn